index.js

```diff
@@ -65,7 +65,11 @@ function reproject(srcpath, dstpath) {
   options.dst.geoTransform = info.geoTransform;
   options.dst.srs = options.t_srs;
 
-  gdal.reprojectImage(options);
+  try {
+    gdal.reprojectImage(options);
+  } catch (err) {
+    throw new Error('GDAL Reprojection Error');
```

Comment:
How about doing `throw new Error('GDAL Reprojection Error: ' + err.message);` such that the original message is not muffled. I feel like not muffling here is ideal since then users of this library would still be able to get access to the raw node-gdal exception details. This feels valuable since this library is low level. Then over at https://github.com/mapbox/preprocessorcerer/pull/88 you could check if the error contains `GDAL Reprojection Error` rather than checking if it is exactly `GDAL Reprojection Error`.

Reply:
thanks @springmeyer! great suggestion. will get this going next week with @mapsam.